app/routes/app.jsx

```diff
@@ -1,11 +1,13 @@
-import { json } from "@remix-run/node";
-import { Link, Outlet, useLoaderData, useRouteError } from "@remix-run/react";
+import { Outlet, useLoaderData, useRouteError } from "react-router";
 import polarisStyles from "@shopify/polaris/build/esm/styles.css?url";
+import { AppProvider as PolarisAppProvider } from "@shopify/polaris";
```

Comment:
The Remix package abstracted this away in it's AppProvider.  The React Router Template no longer does this because it wants partners to use Polaris Web Components instead of Polaris React.

But, components provided by `@shopify/discount-app-components` did not look good with Polaris Web Components.  So, I decided to run both Polaris React for the custom pages, and Polaris Web Componets for the standard app template pages.

This is why we have both providers and a little more setup code in this file